### ISSUE TITLE
WP Admin lockout when WC Admin is active when WC is not

### ISSUE THREAD

**Issue by jacoswan** [woocommerce/woocommerce-admin#2373]
**Describe the bug**
When WooCommerce Admin is active and WooCommerce isn't, you're immediately locked out of WP Admin with this message: 

> The site is experiencing technical difficulties. Please check your site admin email inbox for instructions.

You do receive this email:

```
Howdy!

Since WordPress 5.2 there is a built-in feature that detects when a plugin or theme causes a fatal error on your site, and notifies you with this automated email.

In this case, WordPress caught an error with one of your plugins, WooCommerce Admin.

First, visit your website (https://your.website/) and check for any visible issues. Next, visit the page where the error was caught (https://your.website/wp-admin/plugins.php?deactivate=true&plugin_status=all&paged=1&s=) and check for any visible issues.

Please contact your host for assistance with investigating this issue further.

If your site appears broken and you can't access your dashboard normally, WordPress now has a special "recovery mode". This lets you safely login to your dashboard and investigate further.

https://your.website/wp-login.php?action=enter_recovery_mode&rm_token=nszl3FZUKeITiivH7wQuWy&rm_key=45mLebzGnZodRLOJqLz3DK

To keep your site safe, this link will expire in 1 day. Don't worry about that, though: a new link will be emailed to you if the error occurs again after it expires.

 Error Details
 An error of type E_ERROR was caused in line 137 of the file /srv/htdocs/wp-content/plugins/woocommerce-admin/woocommerce-admin.php. Error message: Uncaught Error: Class 'WC_Admin_Loader' not found in /srv/htdocs/wp-content/plugins/woocommerce-admin/woocommerce-admin.php:137
 Stack trace:
 0 /wordpress/core/5.2.1/wp-includes/class-wp-hook.php(286): wc_admin_devdocs('')
 1 /wordpress/core/5.2.1/wp-includes/class-wp-hook.php(310): WP_Hook->apply_filters(NULL, Array)
 2 /wordpress/core/5.2.1/wp-includes/plugin.php(465): WP_Hook->do_action(Array)
 3 /wordpress/core/5.2.1/wp-admin/includes/menu.php(154): do_action('admin_menu', '')
 4 /wordpress/core/5.2.1/wp-admin/menu.php(309): require_once('/wordpress/core…')
 5 /wordpress/core/5.2.1/wp-admin/admin.php(152): require('/wordpress/core…')
 6 /wordpress/core/5.2.1/wp-admin/plugins.php(10): require_once('/wordpress/core…')
 7 {main}
 thrown
```

And it will take a click on the 3rd link to be able to get back into WP Admin, where you might see a message under WC Admin, if you're looking for it. 

**To Reproduce**

1. Active WooCommerce Admin when WooCommerce is not active, or
2. Deactivate WooCommerce when both plugins are active.

Similar, but different to #1752 and #1805 

**Expected behavior**
I would have expected a soft landing, like a subtle message in the WP Admin area saying the main WooCommerce plugin is needed for this one to work. 

**Screenshots**

When you're booted out of WP Admin:
![Error Message](https://cld.wthms.co/4Mlz27+)

When you get back in via the Recovery link from the email:
![Recovery message](https://cld.wthms.co/VNUxe7+)

**Desktop (please complete the following information):**

* OS: Mac OS Mojave 10.14.5
* Browser Chrome 74.0.3729.169 also Firefox 67.0.1

**Additional context**

@mattyza

**Comment by mattyza** [comment]
Thanks @jacoswan ! cc @timmyc

**Comment by timmyc** [comment]
Thanks for the bug report @jacoswan - looks like this bug was fixed in #2226 and is no longer present in master. We should hopefully have a new release out with this fix included later this week.
